pkg/provider/data_source_regions_test.go

```diff
@@ -0,0 +1,52 @@
+package provider
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+func TestAccDataSourceRegions_basic(t *testing.T) {
+	var (
+		acc_env_vars_checklist = []string{
+			"BA_TF_ACC_VAR_datasource_regions_project_id",
+			"BA_TF_ACC_VAR_datasource_regions_provider",
+			"BA_TF_ACC_VAR_datasource_regions_region_id",
+			"BA_TF_ACC_VAR_datasource_regions_status",
```

Comment:
Please add those newly added ACC Test env vars to the `.env.example` file.

Reply:
For the environment variable names, I'd suggest going with e.g. `BA_TF_ACC_VAR_regions_project_id`. We can skip the `_datasource_`. 

This is how we did for the PGD Data source test. 
If you would like to keep the `_datasource_` in the acceptance test env var names, let's update other data source env var names as well (preferably in a separate PR)

Reply:
I will remove `datasource_` part